src/java/fr/paris/lutece/plugins/userassignment/business/ResourceUser.java

```diff
@@ -65,12 +65,12 @@ public int getId( )
     }
 
     /**
-     * @param _id
+     * @param id
      *            the _id to set
```

Comment:
The param names should also be changed in their descriptions in this class

Reply:
Done